homeassistant/components/shopping_list/const.py

```diff
@@ -1,5 +1,6 @@
 """All constants related to the shopping list component."""
 DOMAIN = "shopping_list"
+EVENT = "shopping_list_updated"
```

Comment:
Let's give this a better name
```suggestion
EVENT_SHOPPING_LIST_UPDATED = "shopping_list_updated"
```

Reply:
Done